src/env_node.ml

```diff
@@ -123,6 +125,22 @@ let rec ocaml_flags t ~profile ~expander =
     t.ocaml_flags <- Some flags;
     flags
 
+let inline_tests t ~profile =
+  match t.inline_tests with
+  | Some x -> x
+  | None ->
+    let state : Dune_env.Stanza.Inline_tests.t =
+      match find_config t ~profile with
+      | None | Some {inline_tests = None; _} ->
+        if profile = "release" then
+          Disabled
```

Comment:
That doesn't seem right to me. If the config is absent, then we should default to the parent node rather than disabling the tests. See how `inherit_from` is used in other functions for an example.